readthedocs/settings/dev.py

```diff
@@ -57,7 +57,6 @@ def DATABASES(self):  # noqa
     def LOGGING(self):  # noqa - avoid pep8 N802
         logging = super().LOGGING
         logging['handlers']['console']['level'] = 'DEBUG'
-        logging['formatters']['default']['format'] = '[%(asctime)s] ' + self.LOG_FORMAT
```

Comment:
I'm guessing we'll have other customizations like this in production. I think we need to be careful here, since we don't want to break syslog, and I think it depends on the content/structure of the messages.

Reply:
We can keep using (r)syslog together with New Relic, we just need to use a structlog's processor compatible with it. Basically, one that keeps things in plain text as they are right now. It shouldn't be a problem.